main.tf

```diff
@@ -0,0 +1,69 @@
+locals {
+  TAGS = merge({ Name = "${var.name}-privatelink" }, var.tags)
+}
+
+// Network Load Balancer to target the service
```

Comment:
It's not really common to put these comments in here, so I really prefer to omit them. And if you do really really want them in here, then make sure the comments are consistent. You are now using `//`, `#` and `##` and something start with a capital and sometimes not.

But again, I think it's better to omit the info here and update the README instead if you need to add info/details.

Reply:
I will add it to the README 👍